edk2toollib/utility_functions.py

```diff
@@ -166,7 +166,7 @@ def wrap(*args):
 # @return returncode of called cmd
```

Comment:
Need a function header comment for the variable...but otherwise looks good.

Reply:
Thanks, added.